### PR TITLE
Pcg 15 add footer

### PR DESCRIPTION
**Pull request by irinamishchenko**
- created footer with navigation, information about project and logo
- added it to App.tsx
- added styles

### CODE REVIEW

File: src/components/footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  Toolbar,
+  BottomNavigation,
+  BottomNavigationAction,
+  Typography,
+} from '@mui/material';
+import Logo from '../../icons/Logo';
+
+const Footer = () => {
+  const [value, setValue] = useState<number>(0);
+  const footerText: string =
```
Comment (makandriival) [inline src/components/footer/Footer.tsx:15]: you can keep all the constants in a separate folder, just to keep the files cleaner.
I know that you might argue that you are using it only once but i would take it out to constants folder still 😊

**Review by makandriival — APPROVED** [review]
will be honest i didn't run it locally.
the code LGTM
good job 😁

**Review by KolyaIvankov22 — APPROVED** [review]

File: src/components/footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    'Your virtual garden assistant. Add your favourite plants, identify it with photo, find care tips and get reminders for watering and other useful events.';
+
+  const bottomNavigationActionStyles = {
+    '& .MuiBottomNavigationAction-label': {
+      color: '#fff',
+      fontSize: '20px',
```
Comment (ewa-wwg) [inline src/components/footer/Footer.tsx]: would be better to use rem instead of px

File: src/components/footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      paddingBottom: '5px',
+    },
+  };
+
+  return (
+    <Box
+      component="footer"
+      sx={{
+        // position: 'stiky',
+        // bottom: '0',
```
Comment (ewa-wwg) [inline src/components/footer/Footer.tsx]: remove the commented code :)

**Review by ewa-wwg — APPROVED** [review]
In general I would try to use rem for font sizes, margins, paddings, but it's a small thing, you don't have to change it now - more for the future.

**Pull request merged**